src/Boost.sol

```diff
@@ -1,29 +1,55 @@
-// SPDX-License-Identifier: MIT
+/**
+ * SPDX-License-Identifier: MIT
+ *
+ *   ____                      _
+ *  |  _ \                    | |
+ *  | |_) |  ___    ___   ___ | |_
+ *  |  _ <  / _ \  / _ \ / __|| __|
+ *  | |_) || (_) || (_) |\__ \| |_
+ *  |____/  \___/  \___/ |___/ \__|
+ */
+
 pragma solidity ^0.8.15;
 
 import "openzeppelin-contracts/access/Ownable.sol";
 import "openzeppelin-contracts/utils/cryptography/SignatureChecker.sol";
 import "openzeppelin-contracts/utils/cryptography/draft-EIP712.sol";
 import "openzeppelin-contracts/token/ERC721/extensions/ERC721URIStorage.sol";
-
 import "./IBoost.sol";
 
+/**
+ * @title Boost
+ * @author @SnapshotLabs - orlando@snapshot.org
```

Comment:
should we put admin@snapshot.org ? or maybe create dev@snapshot.org? What happens if you're not at snapshot anymore?

Reply:
yeah ill put admin@snapshot.org. feel like people would just join the discord or reach out on twitter if they want to ask a q anyway